Service/ConsumerWrapper.php

```diff
@@ -2,10 +2,14 @@
 
 namespace Happyr\DeferredEventSimpleBusBundle\Service;
 
+use Happyr\DeferredEventSimpleBusBundle\Traits\LoggerTrait;
 use SimpleBus\Asynchronous\Consumer\SerializedEnvelopeConsumer;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
 
-class ConsumerWrapper
+class ConsumerWrapper extends ContainerAwareCommand
```

Comment:
We should avoid ContainerAware*
It makes it more difficult to test. This class should not extend anything